web/Controllers/AdminController.cs

```diff
@@ -95,4 +95,36 @@ private void LoggerAlreadyAdminWarning(ILogger logger, BricksAndHeartsUser user)
                 "danger",
                 "Already an admin"));
     }
+
+    public async Task<IActionResult> PropertyList()
+    {
+        var propertyList = await _adminService.GetPropertyList();
+        var listOfProperties = propertyList.Select(PropertyViewModel.FromDbModel).ToList();
+        return View("AllProperties", new PropertiesDashboardViewModel(listOfProperties));
+    }
+
+    public async Task<IActionResult> ViewLandlord(int landlordId)
+    {
+        var landlord = await _adminService.GetLandlordDbModelFromId(landlordId);
+        if (landlord == null)
+        {
+            return StatusCode(404);
+        }
+
+        var LandlordViewModel = LandlordProfileModel.FromDbModel(landlord);
+        return View("Profile", LandlordViewModel);
+    }

```

Comment:
Can we link to the regular landlord profile on the LandlordController? It would be nice if we can avoid duplicating this and Profile.cshtml